components/console/commands.c

```diff
@@ -184,27 +169,32 @@ static const cmd_item_t *find_command_by_name(const char *name)
 
 esp_err_t esp_console_run(const char *cmdline, int *cmd_ret)
 {
-    if (s_tmp_line_buf == NULL) {
-        return ESP_ERR_INVALID_STATE;
```

Comment:
Probably still need to return ESP_ERR_INVALID_STATE if esp_console_init wasn't called?